Vikta-API-testing-sample/src/test/java/com/test/api/category/CategoryTest.java

```diff
@@ -57,9 +69,228 @@ public void checkCategoryInAllCategoriesList() {
                 .extract()
                 .as(CategoryDTO[].class);
 
-        assertThat("Category wasn't found in all categories list",
-                Arrays.asList(categories),
-                hasItem(equalTo(expectedCategory)));
+        assertThat(categories).as(() -> "Category list shouldn't be empty").isNotEmpty();
+
+        for(CategoryDTO category : categories){
```

Comment:
Assertions in a loop should be avoided. Find an assertion that checks the content of the list or use SoftAssertions.